src/services/ceramic-service.ts

```diff
@@ -27,7 +27,8 @@ export default class CeramicServiceImpl implements CeramicService {
   }
 
   async loadDocument<T extends Doctype>(docId: DocID): Promise<T> {
-    return await this._client.loadDocument(docId, {sync: false})
+    const docPromise = this._client.loadDocument(docId, {sync: false})
+    const timeoutPromise = new Promise((_, reject) => setTimeout(() => reject(`Timed out loading docid: ${docId.toString()}`), 60 * 1000))
```

Comment:
How does the failure manifest if the timeout fires and rejects the promise? Is an exception thrown?

Reply:
yes